.gitignore

```diff
@@ -2,4 +2,4 @@
 /.secrets
 /dev.html
 /node_modules
-
+yarn-error.log
```

Comment:
it's better to use `*.log` entry here: this form will ignore any logs (like `npm-debug.log`)